.github/workflows/lock.yml

```
@@ -6,6 +6,9 @@ on:
 
 jobs:
   lock:
+    permissions:
+      issues: write  # for dessant/lock-threads to lock issues
+      pull-requests: write  # for dessant/lock-threads to lock PRs
```

It the only thing in this workflow, so a bit redundant.

```suggestion
      issues: write
      pull-requests: write
```